controllers/database_controller.go

```diff
@@ -492,6 +493,11 @@ func (r *DatabaseReconciler) reconcilePSMDB(ctx context.Context, req ctrl.Reques
 			psmdb.Spec.PMM.Image = database.Spec.Monitoring.PMM.Image
 		}
 		if database.Spec.Backup != nil {
+			backupType := pbm.LogicalBackup
+			if database.Spec.Backup.Type == pbm.PhysicalBackup {
```

Comment:
We need to check the version of the operator because we can have 1.13 installed and it'll break cluster